docs/docs/troubleshooting.md

```diff
@@ -0,0 +1,33 @@
+---
+id: troubleshooting
+title: Troubleshooting
+sidebar_title: Troubleshooting
+---
+### Summary
+
+The following page provides suggestions for common errors that may occur during firmware compilation. If the information provided is insufficient to resolve the issue, feel free to seek out help from the [ZMK Discord](https://discord.gg/VJnx9kr).
+
+### Environment Variables not Defined
+
+An error along the lines of `CMake Error at (zmk directory)/zephyr/cmake/generic_toolchain.cmake:64 (include): include could not find load file:` during firmware compilation indicates that the Zephyr Environment Variables are not properly defined.
+For more information, click [here](../docs/dev-setup#environment-variables).
```

Comment:
Don't use a relative URL here, in case this document moves to a different location in the future:

```suggestion
For more information, click [here](/docs/dev-setup#environment-variables).
```